test/libsolidity/SolidityEndToEndTest.cpp

```diff
@@ -2815,9 +2815,17 @@ BOOST_AUTO_TEST_CASE(include_creation_bytecode_only_once)
 		}
 	)";
 	compileAndRun(sourceCode);
+
+	auto const& output = m_compiler.output();
+	auto const* contractDouble = output.contract("Double");
+	auto const* contractSingle = output.contract("Single");
+
+	solAssert(contractDouble);
+	solAssert(contractSingle);
+
 	BOOST_CHECK_LE(
-		double(m_compiler.object("Double").bytecode.size()),
-		1.2 * double(m_compiler.object("Single").bytecode.size())
+		double(contractDouble->object.size()),
+		1.2 * double(contractSingle->object.size())
```

Comment:
```suggestion
		static_cast<double>(contractDouble->object.size()),
		1.2 * static_cast<double>(contractSingle->object.size())
```

I know it's not something you wrote, but when already touching the line, might as well :)